pkg/apis/serving/v1alpha2/framework_custom.go

```diff
@@ -46,6 +46,7 @@ func (c *CustomSpec) CreateExplainerContainer(modelName string, parallelism int,
 }
 
 func (c *CustomSpec) ApplyDefaults(config *InferenceServicesConfig) {
+    c.Container.Name = constants.InferenceServiceContainerName
```

Comment:
looks like not formatted and aligned with following lines

Reply:
formatted